autonomous/rotate.py

```diff
@@ -9,5 +9,7 @@ class Rotate(AutonomousStateMachine):
 
     @state(first=True)
     def rotate(self):
-        if self.drivetrain.rotate(degrees=-90):
+        if self.drivetrain.rotate(degrees=-90):     
+            '''This tells us that if the robot rotates 90 degrees, the state is done.
+            it calls on the drivetrain and it's code for "rotate", then if it can run it, the state ends.'''
```

Comment:
Perhaps this string should be moved up to the `rotate` function (just below), and then mention that the function depends on the state of `degrees`?

Reply:
The rotate function does not return whether it can run it, it returns if it is done turning